posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -48,82 +49,14 @@ def select(self):
             ),
             ast.Alias(alias="last_seen", expr=ast.Call(name="max", args=[ast.Field(chain=["timestamp"])])),
             ast.Alias(alias="first_seen", expr=ast.Call(name="min", args=[ast.Field(chain=["timestamp"])])),
-            ast.Alias(
-                alias="description",
-                expr=ast.Call(
-                    name="nullIf",
-                    args=[
-                        ast.Call(
-                            name="coalesce",
-                            args=[
-                                self.extracted_exception_list_property("value"),
-                                ast.Call(name="any", args=[ast.Field(chain=["properties", "$exception_message"])]),
-                            ],
-                        ),
-                        ast.Constant(value=""),
-                    ],
-                ),
-            ),
-            ast.Alias(
-                alias="exception_type",
-                expr=ast.Call(
-                    name="nullIf",
-                    args=[
-                        ast.Call(
-                            name="coalesce",
-                            args=[
-                                self.extracted_exception_list_property("type"),
-                                ast.Call(name="any", args=[ast.Field(chain=["properties", "$exception_type"])]),
-                            ],
-                        ),
-                        ast.Constant(value=""),
-                    ],
-                ),
-            ),
+            ast.Alias(alias="id", expr=ast.Field(chain=["properties", "$exception_issue_id"])),
```

Comment:
Again this should be coming from the overrides table? Eventually I mean, not necessarily in this PR.

Reply:
Yep, exactly. This is where we will `coalesce` any value from the overrides table with the value on the event